test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -18,13 +18,13 @@
 TF_ADDONS_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "testTFAddons")
 TF_DATASERVICE_TEST_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "testDataservice")
 
-TF_EC2_GPU_INSTANCE_TYPE = get_ec2_instance_type(default="p2.8xlarge", processor="gpu")
+TF_EC2_GPU_INSTANCE_TYPE = get_ec2_instance_type("g3.16xlarge", "gpu")
 TF_EC2_CPU_INSTANCE_TYPE = get_ec2_instance_type(default="c5.4xlarge", processor="cpu")
 
 
 @pytest.mark.integration('tensorflow_sanity_test')
 @pytest.mark.model("N/A")
-@pytest.mark.parametrize("ec2_instance_type", TF_EC2_GPU_INSTANCE_TYPE, indirect=True)
+@pytest.mark.parametrize("ec2_instance_type", get_ec2_instance_type("g3.16xlarge", "gpu"), indirect=True)
```

Comment:
why can't this be TF_EC2_GPU_INSTANCE_TYPE?

Reply:
Reverted this change.